.github/workflows/remote-build-android-hybrid.yml

```diff
@@ -0,0 +1,30 @@
+name: Remote Build Android Hybrid
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest-xl
+    steps:
+      - name: Checkout
+        # v4
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608
+        with:
+          submodules: true
+          ref: ${{ github.event.pull_request.head.sha }}
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+
+      - name: Setup NodeJS
```

Comment:
:eye_twitch:

```suggestion
      - name: Setup Node.js
```

Reply:
haha how about we unify it to `Setup Node` as per most of the workflows in the repo? I'll commit this update, please feel free to push this suggestion afterwards if that's what you prefer!